src/managers/organization.js

```diff
@@ -121,8 +121,9 @@ export const joinOrganization = async (siret, user_id, is_external) => {
 
   const user_label =
     !given_name && !family_name ? email : `${given_name} ${family_name}`;
-  const usersInOrganizationAlreadyWithoutExternal =
-    usersInOrganizationAlready.filter(({ is_external }) => !is_external);
+  const usersInOrganizationAlreadyWithoutExternal = usersInOrganizationAlready.filter(
```

Comment:
it seems that we are running different linting rules... 🤔

Reply:
or different prettier version ?

Reply:
what bothers me is that prettier is ok in both cases